### PR TITLE
New version of pygithub is raising a different Exception class when a deleted branch does not exist

### PR DESCRIPTION
**Pull request by macdiesel**
@edx/pipeline-team @cpennington PTAL.

### CODE REVIEW

**Review by doctoryes — COMMENTED** [review]
Same create/delete_branch calls exist in create_release_candidate.py also. Perhaps change the exception catching there as well?

**Review by cpennington — APPROVED** [review]
It seems like there might be something subtler going on here (or some other way we could catch it before that particular call), but for now, let's get the release unblocked.

File: scripts/cut_branch.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """
 Command-line script to create a release candidate for an application
 """
+# pylint: disable=broad-except
```
Comment (cpennington) [inline scripts/cut_branch.py]: It might be better to put this on a per-line basis, so that other spots don't slip through.

**Review by cpennington — APPROVED** [review]

**Comment by macdiesel** [comment]
@doctoryes comments addressed.

File: scripts/cut_branch.py (hunk 1 above)
Comment (macdiesel) [reply scripts/cut_branch.py]: done.

**Pull request merged**